single-hit-redirect/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Single Hit Redirect Demo</title>
+    <link rel="stylesheet" href="../demo-styles.css">
+
+    <script src="../bower_components/ff-web-components/dist/vendor/custom-elements-es5-adapter.js"></script>
+    <script src="../bower_components/ff-web-components/dist/vendor/webcomponents-loader.js"></script>
+    <script defer src="../bower_components/ff-web-components/dist/bundle.js"></script>
+    <script>
+        document.addEventListener("WebComponentsReady", function () {
+            factfinder.communication.ResultDispatcher.subscribe('result', function(result) {
+                const storage = factfinder.common.localStorage;
+
+                if (storage.getItem('ff_redirect_off') !== '1' || storage.getItem('ff_redirect_referer') !== window.location.href) {
+                    if (result.resultArticleNumberStatus === 'resultsFound' && result.records && result.records.length === 1) {
```

Comment:
It should be `result.resultStatus` I think

Reply:
As far as i know, this field gets value `resultsFound` if response has at least one record so it will redirects almost all the time. `resultArticleNumberStatus ` gets value `resultsFound` only if one product is returned and its similarity was 100%

Reply:
Ah, ok didnt know. I didnt get that attribute for some reason, I can change it back